### DISCUSSION TITLE
English version

### DISCUSSION THREAD

**Discussion by ALPERDURUKAN** [mushan0x0/AI0x0.com#483] [Ideas]
It seems like a fantastic app, please also provide an English translation!

**Comment by mushan0x0** [comment]
Thank you, the English version will be updated in the future, stay tuned!

**Reply by ALPERDURUKAN** [reply]
Thank you, eagerly waiting for it!

**Comment by beradiant** [comment]
Is there an English version yet? It has been 3 years since this discussion and a similar issue was raised.

**Reply by mushan0x0** [reply]
Thank you for your attention. The English version will be released in the new application in the coming months, featuring enhanced intelligence and improved user experience.
